### PR TITLE
Action wrapper and allure attachment support

### PR DESCRIPTION
**Pull request by dtinth**
## Problems

- In a browser test, it would be great if we could take a screenshot after each step.

- In API test it would be great if we could log all the request and response and attach it to the Allure report so that we can inspect it later.

## Solution

- Add `context.attach(name, buffer, mimeType)` to attach binary attachment to each step.

- Add `actionWrapper` to allow running code before/after each step.

## Screenshot

TBA

### CODE REVIEW

File: docs/guide/api.md [hunk 1]
```diff
@@ -46,6 +46,9 @@ with these arguments:
 * **context** - The context object contains:
   * `log(...)` - Logs a message to the console. Use this instead of
     `console.log()` so that it doesn’t mess up console output.
+  * `attachment(name, buffer, mimeType)` — Attachs some binary output. For
```
Comment (zugarzeeker) [inline docs/guide/api.md]: 👍

**Comment by dtinth** [comment]
This is now being used on production, PR will be merged now.

**Comment by Cerberus** [comment]
@dtinth Have you planned to release a new one ? : )

**Comment by dtinth** [comment]
Sorry, I didn’t catch this message. It’s already released as `v0.55555.5-555` (you can see the release commit as the one before I merge).

To reduce confusion, I will release a new version `v0.555555.0` which is identical to `v0.55555.5-555`.

**Pull request merged**